roles/kubernetes/node/templates/kubelet.j2

```diff
@@ -13,7 +13,7 @@ KUBELET_HOSTNAME="--hostname-override={{ ansible_hostname }}"
 --pod-infra-container-image={{ pod_infra_image_repo }}:{{ pod_infra_image_tag }} \
 --kube-reserved cpu={{ kubelet_cpu_limit }},memory={{ kubelet_memory_limit|regex_replace('Mi', 'M') }} \
 --node-status-update-frequency={{ kubelet_status_update_frequency }} \
-{% if kube_version | version_compare('v1.6', '>=') %}
+{% if kube_version | version_compare('v1.7', '>=') %}
 --enable-cri={{ kubelet_enable_cri }} --cgroups-per-qos={{ kubelet_cgroups_per_qos }} \
```

Comment:
interesting. i just tested it and kubelet service failed to start with 'Error: unknown flag: --enable-cri'. maybe you haven't changed the version to 1.7.0? because the check just excludes the block from being used if your version is still 1.6.* and on top of that it does not make sense to me to change the check like that. i think it should be something like 
```
{% if kube_version | version_compare('v1.6', '>=') %}
{# flag got removed with 1.7.0 #}
{% if kube_version | version_compare('v1.7', '<') %}
--enable-cri={{ kubelet_enable_cri }}
{% endif %}
```
what does `kubectl version` say on your cluster.